holoviews/plotting/bokeh/raster.py

```diff
@@ -161,6 +164,239 @@ def get_data(self, element, ranges, style):
                        'height': 1, 'width': 1}, style)
 
 
+
+class RadialHeatMapPlot(CompositeElementPlot, ColorbarPlot):
+
+
+    start_angle = param.Number(default=np.pi/2, doc="""
+        Define starting angle of the first annulars. By default, beings at 
+        12 o clock.""")
+
+    padding_inner = param.Number(default=0.1, bounds=(0, 0.5), doc="""
+        Define the radius fraction of inner, empty space.""")
+
+    padding_outer = param.Number(default=0.05, bounds=(0, 1), doc="""
+        Define the radius fraction of outer space including the labels.""")
+
+    show_nth_label = param.Number(default=1, doc="""
+        Define every nth label to be plotted. By default, every label is
+        shown.""")
+
+    separate_nth_segment = param.Number(default=0, doc="""
+        Add separation lines between segments for better readability. By
+        default, does not show any separation lines.""")
+
+    # Force x and y ranges to be numerical
+    _y_range_type = Range1d
+    _x_range_type = Range1d
+
+    # Map each glyph to a style group
+    _style_groups = {'annular_wedge': 'annular',
+                     'text': 'label',
+                     'multi_line': 'separator'}
+
+    _draw_order = ["annular_wedge", "multi_line", "text"]
+
+    style_opts = (['separator_' + p for p in line_properties] + \
+                  ['annular_' + p for p in fill_properties + line_properties] + \
+                  ['label_' + p for p in text_properties] + ['width', 'cmap'])
+
+
+    @staticmethod
+    def _extract_implicit_order(array):
```

Comment:
I think you can use ``holoviews.core.util.unique_array`` or ``holoviews.core.util.unique_iterator`` for this instead.

Reply:
Thanks for the hint - no need to reimplement what is already there :-)